include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -296,6 +299,45 @@ SECTIONS
 
     GROUP_END(RAMABLE_REGION)
 
+#ifdef CONFIG_CCM_BASE_ADDRESS
+
+    GROUP_START(CCM)
+
+	SECTION_PROLOGUE(_CCM_BSS_SECTION_NAME, (NOLOAD OPTIONAL),SUBALIGN(4))
+	{
+		__ccm_start = .;
+		__ccm_bss_start = .;
+		*(.ccm_bss)
+		KEEP(*(SORT_BY_NAME(".ccm_bss.*")))
+	} GROUP_LINK_IN(CCM)
+
+	__ccm_bss_end = .;
+
+	SECTION_PROLOGUE(_CCM_NOINIT_SECTION_NAME, (NOLOAD OPTIONAL),SUBALIGN(4))
+	{
+		__ccm_noinit_start = .;
+		*(.ccm_noinit)
+		KEEP(*(SORT_BY_NAME(".ccm_noinit.*")))
```

Comment:
do we need these KEEP()s here? why are we sorting them? This seems like copypasta.

Reply:
@andrewboie to be honest me ld.scripting knowledge is very limited, so yes it mainly is copy/paste from other sections with comparable functionality.  I just now tried with; 

`*(".ccm_data.*")`

and that seems to work fine too.  Do you want me to remove the KEEP()/SORT() ?

Reply:
yes.